onnxruntime/core/common/str_helper.cc

```diff
@@ -20,10 +20,10 @@ std::string ToMBString(const std::wstring& s) {
 std::wstring ToWideString(const std::string& s) {
   if (s.size() >= static_cast<size_t>(std::numeric_limits<int>::max())) throw std::runtime_error("length overflow");
   const int src_len = static_cast<int>(s.size() + 1);
-  const int len = MultiByteToWideChar(CP_ACP, 0, s.data(), src_len, nullptr, 0);
+  const int len = MultiByteToWideChar(CP_UTF8, 0, s.data(), src_len, nullptr, 0);
```

Comment:
Would it be an issue to always expect UTF8?

Reply:
Currently this helper is used in loading model files on the c-abi, and the adapter. CP_ACP should not be used per the documentation, as it relied on the currenly set default windows code page. It can change based on user settings.

Instead it should take in UTF8 which will ensure consistent behavior.

ASCII is a subset of UTF8, so all ascii filenames will work as is.

Reply:
Should ToMBString and the comment at https://github.com/microsoft/onnxruntime/pull/3872/files#diff-576bfb1861f96bba0973d65fb2e6ab9aL233 be changed too?

Reply:
I saw that and kept it the same, as it is used primarily as a way to update the ort status strings. I didn't want to change that format until discussing with others, as customers could have potentially taken an erroneous dependency on the default windows code page.